.github/workflows/containers.yml

```diff
@@ -101,7 +101,7 @@ jobs:
                 env:
                     APIML_SECURITY_OIDC_CLIENTID: ${{ secrets.OKTA_CLIENT_ID }}
                     APIML_SECURITY_OIDC_CLIENTSECRET: ${{ secrets.OKTA_CLIENT_PASSWORD }}
-                    APIML_SECURITY_OIDC_INTROSPECTENDPOINT: /v1/introspect
+                    APIML_SECURITY_OIDC_INTROSPECTURL: https://dev-95727686.okta.com/oauth2/default/v1/introspect
```

Comment:
I am wondering whether this should not go to secrets

Reply:
Good point. I'll move it to Github secrets.

Reply:
Done